test/common/testinfra.py

```diff
@@ -55,28 +55,31 @@
 
 BASELINE_PRIORITY = 10
 
+BRANCHES = [ 'master', 'rhel-7.4', 'rhel-7.3.1', 'rhel-7.3.2', 'rhel-7.3.3', 'rhel-7.3.4', 'rhel-7.3.5' ]
+
 DEFAULT_VERIFY = {
-    'avocado/fedora-24': [ 'master', 'pulls' ],
+    'avocado/fedora-24': BRANCHES,
     'avocado/fedora-25': [ ],
-    'container/kubernetes': [ 'master', 'pulls' ],
+    'container/kubernetes': BRANCHES,
     'koji/fedora-24': [ ],
     'koji/fedora-25': [ ],
     'koji/fedora-26': [ ],
-    'selenium/firefox': [ 'master', 'pulls' ],
-    'selenium/chrome': [ 'master', 'pulls' ],
-    'verify/centos-7': [ 'master', 'pulls' ],
-    'verify/continuous-atomic': [ 'master' ],
-    'verify/debian-stable': [ 'master', 'pulls', ],
-    'verify/debian-testing': [ 'master', 'pulls' ],
+    'selenium/firefox': BRANCHES,
+    'selenium/chrome': BRANCHES,
+    'verify/centos-7': BRANCHES,
+    'verify/continuous-atomic': [ ],
+    'verify/debian-8': [ 'rhel-7.4' ],
+    'verify/debian-stable': [ 'master' ],
+    'verify/debian-testing': BRANCHES,
     'verify/fedora-24': [ ],
-    'verify/fedora-25': [ 'master', 'pulls' ],
-    'verify/fedora-i386': [ 'master', 'pulls' ],
-    'verify/fedora-26': [ 'master', 'pulls' ],
-    'verify/fedora-atomic': [ 'master', 'pulls' ],
+    'verify/fedora-25': BRANCHES,
+    'verify/fedora-i386': [ 'master', 'rhel-7.4', 'rhel-7.3.5' ],
+    'verify/fedora-26': [ 'master' ],
+    'verify/fedora-atomic': BRANCHES,
     'verify/fedora-testing': [ ],
-    'verify/rhel-7': [ 'master', 'pulls' ],
-    'verify/rhel-atomic': [ 'master', 'pulls' ],
-    'verify/ubuntu-1604': [ 'master', 'pulls' ],
+    'verify/rhel-7': BRANCHES,
+    'verify/rhel-atomic': BRANCHES,
+    'verify/ubuntu-1604': [ 'master', 'rhel-7.4', 'rhel-7.3.5', 'rhel-7.3.4', 'rhel-7.3.3', 'rhel-7.3.2' ],
```

Comment:
I suppose this is to expose backported patches to more variability in dependencies and toolchains, and thus increase the chance to spot some flaw in a backport. Is there a particular reason why you leave out only rhel-7.3.1 here?